client/src/js/modules/types/mx/shipment/views/container-mixin.js

```diff
@@ -321,11 +321,10 @@ export default {
       try {
         this.$store.commit('loading', true)
         const result = await this.$refs.containerForm.validate()
-
         if (result) {
-          await this.saveSample(location)
           const samplesRef = this.$refs.samples
           samplesRef.$refs[`sample-row-${location}`][0].closeSampleEditing()
+          await this.saveSample(location)
```

Comment:
There are two issues here:
1. If the sample can not be saved then the edit row has been closed already
2. It is possible that the await in 323 gives the user a chance to do a second click
I think we should use the loading status to avoid this.

Reply:
See latest commit

Reply:
Yep, that works for me.